NSSAgent/NSSAgent.cs

```diff
@@ -515,17 +520,54 @@ public IQueryable<String> GetRoles()
             return Role.ToList().AsQueryable();
         }
 
+        #endregion
+
+        #region Statuses
+        public IQueryable<Status> GetStatuses()
+        {
+            return this.Select<Status>();
+        }
+        public Task<Status> GetStatus(Int32 ID)
+        {
+            return this.Find<Status>(ID);
+        }
+        public IQueryable<Status> GetApplicableStatuses(Manager manager = null, bool isStreamStats = false)
+        {
+            var query = this.Select<Status>();
+            var env = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+
+            // set applicable statuses for each environment
+            var SSProdStatuses = new List<string> { "SS Approved" };
+            var SSStagingStatuses = new List<string> { "SS Approved", "Review" };
+            var NSSStagingStatuses = new List<string> { "Review", "Approved", "SS Approved" };
+            var NSSProdStatuses = new List<string> { "Approved", "SS Approved" };
```

Comment:
This could probably be done better, I tried to add it to the appsettings.json and had trouble. If there's a good way to do this, let me know.